index.html

```diff
@@ -86,201 +86,247 @@ <h1>
+                        <ul>
+                            <li>
+                                <span class="skills">
+                                    Languages <span><img src="./images/Disabled.png" alt="languages"></span>
                                 </span>
+                                <ul class="skills-languages">
+                                    <li class="add-skills">
+                                        <a href="#"><img src="./images/Ellipse 1.png" alt="JS"></a>
+                                        <span class="text-skills">
+                                            JavaScript
+                                        </span>
+                                    </li>
+                                    <li class="add-skills">
+                                        <a href="#"><img src="./images//Ellipse 1 (1).png" alt="HTML"></a>
+                                        <span class="text-skills">
+                                            HTML
+                                        </span>
+                                    </li>
+                                    <li class="add-skills">
+                                        <a href="#"><img src="./images//Ellipse 1 (2).png" alt="CSS"></a>
+                                        <span class="text-skills">
+                                            CSS
+                                        </span>
+                                    </li>
+                                </ul>
+                            </li>
+                            <li class="skills">Frameworks <span><img src="./images/Enabled.png" alt="frameworks"></span>
+                            </li>
+                            <li>
+                                <hr class="hr-line">
+                            </li>
+                            <li class="skills">Skills <span><img src="./images/Enabled.png" alt="Skills"></span></li>
+                            <li>
+                                <hr class="hr-line">
                             </li>
                         </ul>
-                    </li>
-                    <li class="skills">Frameworks <span><img src="./images/Enabled.png" alt="frameworks"></span></li>
-                     <li>
-                        <hr class="hr-line">
-                     </li>
-                    <li class="skills">Skills <span><img src="./images/Enabled.png" alt="Skills"></span></li>
-                    <li>
-                        <hr class="hr-line">
-                    </li>
-                </ul>
+                    </div>
+
+
+
+                </section>
+
+                <!-- 
+                contact form -->
+
+                <section class="contact-me">
+                    <img class="contact-bg-image" alt="Image missing">
+                    <div class="contact-form-submit">
+                        <h1>
+                            Contact me
+                        </h1>
```

Comment:
- [ ] It is a best practice that webpages have only one `h1` tag on a web page, because of SEO reasons. You already have `h1` tag in the headline section. Please change this one to either `h2` or `h3`.